raiden/routing.py

```diff
@@ -112,7 +112,7 @@ def get_best_routes(
             )
 
             try:
-                route = networkx.shortest_path(
+                route = networkx.shortest_path(  # pylint: disable=E1121
```

Comment:
Why do we get `E1121`, here?

Reply:
The only thing I could find is this old blog post: https://hustoknow.blogspot.com/2013/06/pylint.html

According to that it's a problem in the AST.